lib/handlers/run.js

```diff
@@ -41,11 +42,27 @@ function run(req, res, next, options) {
             blockKeys = _.takeRightWhile(sortedKeys, function(key) {
                 return key === options.startKey;
             });
+
+            if (_.isEmpty(blockKeys)) {
+                log.info('No handlers at next level in New Shard \
+                         continuing at the next available level', {
+                             startKey: options.startKey
```

Comment:
Actually, there is only one known shard at this point, the `startConductor`, but I'll add that in.